packages/conf-binutils/conf-binutils.0.2/files/find-binutils.ml.in

```diff
@@ -58,6 +58,14 @@ let split_results fmt =
     | Some s -> split s in
   ksprintf run fmt
 
+let hd_opt xs = List.nth_opt xs 0
+
+let take_first fmt =
```

Comment:
put all commands as thunks in a list and use `List.find is_some` to find the first that returns `Some x`.